pyproject.toml

```diff
@@ -0,0 +1,148 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel", "setuptools-scm>=8.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "mlx-audio"
+version = "0.2.6"
+description = "MLX-Audio is a package for inference of text-to-speech (TTS) and speech-to-speech (STS) models locally on your Mac using MLX"
+readme = "README.md"
+license = {text = "MIT"}
+authors = [
+    {name = "Prince Canuma", email = "prince.gdt@gmail.com"}
+]
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+requires-python = ">=3.9"
+
+# Core dependencies (minimal for DSP-only usage)
+dependencies = [
+    "mlx>=0.25.2",
+    "numpy>=1.26.4",
+    "huggingface_hub>=0.27.0",
+]
+
+[project.optional-dependencies]
+# STT dependencies
+stt = [
+    "transformers>=4.49.0",
+    "tiktoken>=0.9.0",
+    "tqdm>=4.67.1",
+]
```

Comment:
Needs mistral-common[audio] for voxtral.

I will see how we can remove that dependency later.